### PR TITLE
Left and right types in model relation index and view

### PR DESCRIPTION
**Pull request by didoda**
This provides a readonly view of left and right types in relation modelling index and view.

### CODE REVIEW

File: src/Template/Pages/Model/Relations/index.twig [hunk 1]
```diff
@@ -1 +1,87 @@
...
+
+<div class="module-index">
+    <div class="list-objects">
+
+        {% if resources %}
+            {# list resources : table header #}
+            <nav class="table-header has-border-black">
+                <div class="narrow"><a href="{{ Link.sortUrl('id') }}">{{ __('id') }}</a></div>
+                {% for prop in properties %}
+                    <div class="{{ Link.sortClass(prop) }}"><a href="{{ Link.sortUrl(prop) }}">{{ __(prop) }}</a></div>
```
Comment (DarioSacco) [inline src/Template/Pages/Model/Relations/index.twig]: maybe that `__(prop)` label can be `|humanize`ed? I see underscore notation in some of the props (**Inverse_name** for example)

File: src/Template/Pages/Model/Relations/index.twig [hunk 2]
```diff
@@ -1 +1,87 @@
-{% element 'Model/index_content' { 'title': resourceType|humanize } %}
+{% do _view.assign('title', 'Model ' ~ resourceType|humanize) %}
+{% do _view.assign('bodyViewClass', 'view-module view-model') %}
```
Comment (DarioSacco) [inline src/Template/Pages/Model/Relations/index.twig:2]: I think one `view-model` class is enough 😄

File: src/Template/Pages/Model/Relations/index.twig [hunk 3]
```diff
@@ -1 +1,87 @@
...
+                {% set viewAction = 'model:view:' ~ resourceType %}
+                {% set rowClasses = 'table-row' %}
+                {% if prop == 'enabled' and propVal != 1 %}
+                    {% set rowClasses = rowClasses ~ ' object-status-off' %}
+                {% endif %}
+
+                <a href="{{ Url.build({'_name': viewAction, 'id': resource.id}) }}"
+                    class="{{ rowClasses }}">
+
+                    <div class="narrow"> {{ resource.id }}</div>
```
Comment (DarioSacco) [inline src/Template/Pages/Model/Relations/index.twig]: that extra space before the value can be removed

File: src/Template/Pages/Model/Relations/index.twig [hunk 4]
```diff
@@ -1 +1,87 @@
...
+                        <button
+                            title="{{ __('Edit') }} {{ resource.attributes.name }}"
+                            class="{{ buttonClasses }}">{{ __('Edit') }}</button>
+                    </div>
+
+                </a>
+            {% endfor %}
+
+        {% else %}
+                {{ __('No items found') }}
```
Comment (DarioSacco) [inline src/Template/Pages/Model/Relations/index.twig]: too much indentation here

**Review by DarioSacco — CHANGES_REQUESTED** [review]

File: src/Template/Element/Model/relation.twig [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<div class="input">
+    <label for="">{{__('Left types') }}</label>
+    {% for left in left_object_types|sort %}
+        <span class="tag has-background-module-{{ left }}">{{ left }}</span>
+    {% endfor %}
+</div>
+
+<div class="input">
+    <label for="">{{__('Right types') }}</label>
```
Comment (DarioSacco) [inline src/Template/Element/Model/relation.twig:9]: I don't see translation for this labels. Are they missing in `.po` files?

File: src/Template/Pages/Model/Relations/index.twig (hunk 2 above)
Comment (didoda) [reply src/Template/Pages/Model/Relations/index.twig:2]: `view-module` != `view-model` ;)

File: src/Template/Element/Model/relation.twig (hunk 5 above)
Comment (didoda) [reply src/Template/Element/Model/relation.twig:9]: yes, missing translations for all the modelling part, I think.
I'd rather do a separate PR with all the translations ;)

**Review by DarioSacco — APPROVED** [review]

**Review by DarioSacco — APPROVED** [review]

**Pull request merged**